forge/routes/auth/index.js

```diff
@@ -372,12 +380,12 @@ module.exports = fp(async function (app, opts, done) {
                 request.session.User,
                 'VerifyEmail',
                 {
-                    confirmEmailLink: `${app.config.base_url}/account/verify/${verifyToken}`
+                    confirmEmailLink: `${app.config.base_url}/account/verifyemail/${verifyToken}`
                 }
             )
             reply.send({ status: 'okay' })
         } else {
-            reply.code(400).send({ status: 'error', message: 'Email already verified' })
+            reply.code(400).send({ code: 'invalid_request', error: 'Email already verified' })
```

Comment:
If it's already verified I think it would be better to just redirect to the login page

Reply:
Are you proposing I set the API to send a non 4xx code & then use that in the `vue` to re-direct?
Something return like "202 Accepted" (The request has been received but not acted upon) and simply redirect user to home (which will either show them their home or the login dialog if they are not logged in)